db/model/bot.js

```diff
@@ -94,6 +94,11 @@ module.exports = function bot(seq, dataTypes) {
       allowNull: false,
       comment: 'Bot Version. Use Semantic Versioning ^2.0.0',
     },
+    displayName: {
+      type: dataTypes.STRING,
```

Comment:
should have max length assigned here using one of the constants
... and use the length in swagger regex validation too
... and add tests for max length